langchain/llms/vertexai.py

```diff
@@ -0,0 +1,101 @@
+"""Wrapper around Google VertexAI models."""
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, root_validator
+
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+from langchain.llms.utils import enforce_stop_tokens
+from langchain.utilities.vertexai import (
+    init_vertexai,
+    is_tuned_model,
+    raise_vertex_import_error,
+)
+
+
+class _VertexAICommon(BaseModel):
+    client: Any = None  #: :meta private:
+    model_name: Optional[str] = "text-bison"
+    """Model name to use."""
```

Comment:
It would be more informative to indicate that this could be foundation or tuned at the each class implementation.